src/gui/base/GuiUtils.js

```diff
@@ -125,4 +126,16 @@ export function makeListSelectionChangedScrollHandler(scrollDom: HTMLElement, en
 			scrollDom.scrollTop = selectedBottom - scrollWindowHeight
 		}
 	}
+}
+
+/**
+ * Executes the passed function if the user is allowed to see `tutanota.com` links.
+ * @param render receives the resolved link
+ * @returns {Children|null}
+ */
+export function ifAllowedTutanotaLinks(linkId: InfoLink, render: (string) => Children): Children | null {
```

Comment:
I am wondering if it's better to make a high-level component, something like `HideOnWhitelabel` which doesn't render anything on whitelabel. Maybe using `pureComponent()`? It will do some unnecessary work, sure but it's not a big difference.